cmd/edenPod.go

```diff
@@ -429,10 +429,10 @@ func podInit() {
 	podDeployCmd.Flags().Uint32Var(&appCpus, "cpus", defaults.DefaultAppCPU, "cpu number for app")
 	podDeployCmd.Flags().StringVar(&appMemory, "memory", humanize.Bytes(defaults.DefaultAppMem*1024), "memory for app")
 	podDeployCmd.Flags().StringVar(&diskSize, "disk-size", humanize.Bytes(0), "disk size (empty or 0 - same as in image)")
-	podDeployCmd.Flags().StringVar(&volumeType, "volume-type", "qcow2", "volume type for empty volumes (qcow2, raw or oci); set it to none to not use volumes")
+	podDeployCmd.Flags().StringVar(&volumeType, "volume-type", "qcow2", "volume type for empty volumes (qcow2, raw, qcow, vmdk, vhdx or oci); set it to none to not use volumes")
```

Comment:
While it is pretty awesome that you're trying to cover vmdk and vhdx -- those are not really officially supported in EVE -- so if you encounter any issues I wouldn't spend too much time on those: document them and move on (as in -- feel free to drop it from tests, etc.)